apiary-metastore-listener/README.md

```diff
@@ -51,7 +51,10 @@ The following shows an example JSON message representing an "INSERT" event:
       "tableName": "some_table",
       "files": ["file:/a/b.txt","file:/a/c.txt"],
       "fileChecksums": ["123","456"],
-      "partitionKeyValues": {"load_date": "2013-03-24","variant_code": "EN"}
+      "partitionKeyValues": {
```

Comment:
Won't there be a `partitionKeys` and a `partitionValues` like below?

Reply:
No. It only has partitionKeyValues as a Map.

Reply:
OK, but to be consistent in our messages we could turn their Map back into values and keys then this doesn't become a special case.

Reply:
We don't have type information for partition Keys in Insert Event.

Reply:
We don't have type information for partition Keys in Insert Event.

Reply:
We don't have type information for partition Keys in Insert Event.

Reply:
We don't have type information for partition Keys in Insert Event.

Reply:
We don't have type information for partition Keys in Insert Event.

Reply:
I don't think we can as we don't have type information for the partition keys in the insert Event.